server/src/projects/projects.controller.ts

```diff
@@ -65,12 +65,17 @@ export class ProjectsController {
 
     try {
       if (contactId) {
-        return this.projectsService.findManyByContactId(contactId);
+        return await this.projectsService.findManyByContactId(contactId);
       }
     } catch (e) {
-      const errorMessage = `${e}`;
-
-      throw new HttpException(errorMessage, HttpStatus.BAD_REQUEST);
+      if (e instanceof HttpException) {
+        throw e;
+      } else {
+        throw new HttpException({
+          code: 'PROJECTS_CONTROLLER_UNKNOWN',
```

Comment:
I think this could be worded better — not sure the codes should be named controller-specific, more generic. If you want to mention the controller, it might be done in "detail" or something else.

Reply:
👍  gotcha, will rename